libraries/tools/kotlin-ksp-gradle/build.gradle

```diff
@@ -9,19 +9,26 @@ pill {
 }
 
 dependencies {
-    compile project(':kotlin-gradle-plugin-api')
-    compile project(':kotlin-gradle-plugin-model')
-
-    compile kotlinStdlib()
-    compileOnly project(path: ':kotlin-compiler-embeddable', configuration: 'runtimeJar')
-    compileOnly project(':kotlin-symbol-processing')
```

Comment:
This dependency is dropped, is that intentional?

Reply:
Yes, this is intentional. The gradle plugin depends on neither the compiler nor the compiler plugin. Their interaction is done via command line options.